docs/content/content/using-index-md.md

```diff
@@ -0,0 +1,113 @@
+---
+aliases:
+- /doc/using-index-md/
+lastmod: 2017-02-22
+date: 2017-02-22
+linktitle: Using _index.md
+menu:
+  main:
+    parent: content
+prev: /content/example
+next: /themes/overview
+notoc: true
+title: Using _index.md
+weight: 70
+---
+# \_index.md and 'Everything is a Page'
+
+As of version v0.18 Hugo now treats '[everything as a page](http://bepsays.com/en/2016/12/19/hugo-018/)'. This allows you to add content and frontmatter to any page - including List pages like [Sections](/content/sections/), [Taxonomies](/taxonomies/overview/), [Taxonomy Terms pages](/templates/terms/) and even to 'special case' pages like the [Home page](/templates/homepage/).
+
+In order to take advantage of this behaviour you need to do a few things. 
+
+1. Create an \_index.md file that contains the frontmatter and content you would like to apply.
+
+2. Place the \_index.md file in the correct place in the directory structure. 
+
+3. Ensure that the respective template is configured to display `{{ .Content }}` if you wish for the content of the \_index.md file to be rendered on the respective page. 
+
+## How \_index.md pages work
+
+Before continuing it's important to know that this page must reference certain templates to describe how the \_index.md page will be rendered. Hugo has a multitude of possible templates that can be used and placed in various places (think theme templates for instance). For simplicity/brevity the default/top level template location will be used to refer to the entire range of places the template can be placed. 
+
+If this is confusing or you are unfamiliar with Hugo's template hierarchy, visit the various template pages listed below. You may need to find the 'active' template responsible for any particular page on your own site by going through the template hierarchy and matching it to your particular setup/theme you are using. 
+
+- [Home page template](/templates/homepage/)
+- [Content List templates](/templates/list/)
+- [Single Content templates](/templates/content/)
+- [Taxonomy Terms templates](/templates/terms/)
+
+Now that you've got a handle on templates lets recap some Hugo basics to understand how to use an \_index.md file with a List page.
+
+1. Sections and Taxonomies are 'List' pages, NOT single pages.
+2. List pages are rendered using the template heirarchy found in the [Content - List Template](http://localhost:1313/templates/list/) docs.
+3. The Home page is not technically a list page but has [it's own template](/templates/homepage/) at layouts/index.html.
```

Comment:
Yes, I agree with you but I wanted to capture something of how it can be a 'special case' and inadvertently overlooked how all such pages could have a similar behaviour. I'll reword the doc to clarify.